MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj

```diff
@@ -86,40 +77,18 @@
     <PackageReference Include="Microsoft.NETCore.App" Version="2.1.30" />
     <PackageReference Include="SharpDX" Version="4.0.1" />
     <PackageReference Include="SharpDX.D3DCompiler" Version="4.0.1" />
+    <PackageReference Include="LibKTX" Version="0.9.2" />
```

Comment:
ideally we'd like to have something similar to the `DdsLoader` for the ktx format. But this is good for testing purposes 😄 I'll see if I can dig up the one I started but never had time to finish.

Reply:
Not sure I agree with you on this; the LibKTX is the nuget package for `libktxsharp`, referenced in the Monogame fork of basisU, https://github.com/MonoGame/MonoGame.Tool.BasisUniversal. 

That dll just wasn't available on nuget, until (very strangely timed), I opened a PR to do so, and the maintainer also had a commit ready to rock. https://github.com/mcraiha/libktxsharp/pull/5 

Why would we want to pull in the library without Nuget?

Reply:
In this case we only need to ability to read ktx files, so pulling in a full library might be a bit overkill. 
We can leave this as is for now and see what the other maintainers think when we open the PR on the main repo.

Reply:
The library was mentioned in the BasisU fork because it was an example that I used to test it worked.